app/src/main/java/org/oppia/app/profile/AdminAuthActivityPresenter.kt

```diff
@@ -61,6 +65,18 @@ class AdminAuthActivityPresenter @Inject constructor(
     }
   }
 
+  fun handleOnSavedInstanceState(bundle: Bundle) {
+    bundle.putString(KEY_ADMIN_AUTH_INPUT_ERROR_MESSAGE, authViewModel.errorMessage.get())
+  }
+
+  fun handleOnRestoreInstanceState(bundle: Bundle) {
+    val errorMessage = bundle.getString(KEY_ADMIN_AUTH_INPUT_ERROR_MESSAGE)
+    Log.d("Debug", "Error message = $errorMessage")
```

Comment:
Remove this Log statement. We do not want any log statements in production code.

Reply:
Done